### ISSUE TITLE
Packaging webview-codicons-sample doesn't package the codicons

### ISSUE THREAD

**Issue by Tommycore** [microsoft/vscode-extension-samples#692]
Steps to reproduce:
- Clone repo.
- Open `webview-codicons-sample` folder with VS Code.
- Run `npm i`.
- Install packaging cli via `npm i -g vsce`.
- Remove publisher line from package.json (otherwise you'll get the error message `It's not allowed to use the 'vscode-samples' publisher.`).
- Run `vsce package`.
- Go to the extensions tab and install the freshly created .vsix file.
- Run the command `Cat Codicons: Show Cat Codicons`.

The result (at least for me), was a tab looking like this:
![image](https://user-images.githubusercontent.com/19580006/181626657-238b62d8-dad5-40c3-b7b7-41eff3fab5cf.png)

I opened the `.vsix` file manually and looked inside - and the codicons are nowhere to be found inside. Which begs the question: What am I missing when it comes to packaging? How would I best go about it with a project created with the yeoman generator as described in the official tutorials? Just add `!node_modules/@vscode/codicons/dist/**` to the `.vscodeignore` file?

**Comment by dacort** [comment]
@Tommycore Ever figure this out? Running into the same.

**Comment by Tommycore** [comment]
I can't really remember to be honest. Did adding the line to the `.vscodeignore` file work? If so I probably opened this issue to point out that the documentation is lacking and that I am unsure what the preferred way is.

**Comment by dacort** [comment]
No worries, thanks for the reply! I did try that and it seems to work...specified just the `css` and `ttf` files to keep the size down.

```
!node_modules/@vscode/codicons/dist/codicon.css
!node_modules/@vscode/codicons/dist/codicon.ttf
```

**Comment by phil294** [comment]
Related (maybe also fixes this?) #1005

**Comment by yarivtal** [comment]
Moving npm packaged from devDependencies to dependencies will put a lot of unused files into the packages vsix.
It would have been best if the resulting codicons (ttf, css) were published in a separate npm than the codicons generator app and then THAT npm could be added to dependencies instead of the @vscode/codicons npm package.
